plugins/woocommerce/client/legacy/js/frontend/account-i18n.js

```diff
@@ -0,0 +1,13 @@
+/* global Cookies */
+jQuery( function( $ ) {	
+    // Select all elements with the class 'wc-block-components-notice-banner' that contain text
```

Comment:
```suggestion
    // Select all elements with the [role="alert"] attribute that contain text
```

Reply:
Good catch, made the change here: https://github.com/woocommerce/woocommerce/pull/47144/commits/60e0b972822fd0d2e3ddc68f22699acdde1c8159